<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DIYClass</class>
 <widget class="QWidget" name="DIYClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DIY</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>271</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Â∞èÈ∏ü</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;üê¶&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ark Pixel 12px monospaced zh_cn</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:700;&quot;&gt;Bird&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>371</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="File1Lable">
            <property name="text">
             <string>Êñá‰ª∂ 1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="File1lineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="File1pushButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="File2Lable">
            <property name="text">
             <string>Êñá‰ª∂ 2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="File2lineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="File2pushButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="File3Lable">
            <property name="text">
             <string>Êñá‰ª∂ 3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="File3lineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="File3pushButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="BirdCommitpushButton">
          <property name="text">
           <string>Êèê‰∫§</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BirdClearpushButton">
          <property name="text">
           <string>Ê∏ÖÁ©∫</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>164</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QLabel" name="BirdNameLable">
        <property name="text">
         <string>ÂëΩÂêç</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="BirdNameLineEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>ËÉåÊôØ</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>141</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:72pt;&quot;&gt;üèôÔ∏è&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ark Pixel 12px monospaced zh_cn</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Background</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>381</width>
       <height>60</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="BackgroundLable">
          <property name="text">
           <string>Êñá‰ª∂</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="BackgroundlineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="BackgroundpushButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="BackgroundCommitpushButton">
          <property name="text">
           <string>Êèê‰∫§</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BackgroundClearpushButton">
          <property name="text">
           <string>Ê∏ÖÁ©∫</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Âú∞Èù¢</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>381</width>
       <height>60</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="GroundLabel">
          <property name="text">
           <string>Êñá‰ª∂</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="GroundlineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="GroundpushButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPushButton" name="GroundCommitpushButton">
          <property name="text">
           <string>Êèê‰∫§</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="GroundClearpushButton">
          <property name="text">
           <string>Ê∏ÖÁ©∫</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>141</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:72pt;&quot;&gt;üóª&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ark Pixel 12px monospaced zh_cn</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:700;&quot;&gt;Ground&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>ÁÆ°ÈÅì</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>381</width>
       <height>60</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="PipeLabel">
          <property name="text">
           <string>Êñá‰ª∂</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="PipelineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="PipepushButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QPushButton" name="PipeCommitpushButton">
          <property name="text">
           <string>Êèê‰∫§</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PipeClearpushButton">
          <property name="text">
           <string>Ê∏ÖÁ©∫</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>121</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:72pt;&quot;&gt;ü™£&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ark Pixel 12px monospaced zh_cn</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Pipe</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>ËÉåÊôØÈü≥‰πê</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>381</width>
       <height>60</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="BGMLabel">
          <property name="text">
           <string>Êñá‰ª∂</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="BGMlineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="BGMpushButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QPushButton" name="BGMCommitpushButton">
          <property name="text">
           <string>Êèê‰∫§</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BGMClearpushButton">
          <property name="text">
           <string>Ê∏ÖÁ©∫</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>0</y>
       <width>131</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:72pt;&quot;&gt; üéº&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ark Pixel 12px monospaced zh_cn</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>BGM</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
